From 2d4a9cd04dbfa20f460b8147d03936aa97768158 Mon Sep 17 00:00:00 2001
From: Joel Fernandes <joel@joelfernandes.org>
Date: Sun, 8 Jul 2018 23:07:19 -0700
Subject: [PATCH] DONOT MERGE: Remove version override warnings

Signed-off-by: Joel Fernandes <joel@joelfernandes.org>
---
 src/cc/frontends/clang/loader.cc | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/cc/frontends/clang/loader.cc b/src/cc/frontends/clang/loader.cc
index 81e429032551..53250f426547 100755
--- a/src/cc/frontends/clang/loader.cc
+++ b/src/cc/frontends/clang/loader.cc
@@ -179,8 +179,6 @@ int ClangLoader::parse(unique_ptr<llvm::Module> *mod, TableStorage &ts,
   if (version_override) {
     vmacro = "-DLINUX_VERSION_CODE_OVERRIDE=" + string(version_override);
 
-    std::cout << "WARNING: Linux version for eBPF program is being overridden with: " << version_override << "\n";
-    std::cout << "WARNING: Due to this, the results of the program may be unpredictable\n";
     flags_cstr_rem.push_back(vmacro.c_str());
   }
 
-- 
2.18.0.203.gfac676dfb9-goog

